package/MDAnalysis/coordinates/H5MD.py

```diff
@@ -726,6 +777,635 @@ def __setstate__(self, state):
+
+
+    Parameters
+    ----------
+    filename : str or :class:`h5py.File`
+        trajectory filename or open h5py file
+    n_atoms : int
+        number of atoms in trajectory
+    n_frames : int (optional)
+        number of frames to be written in trajectory
+    driver : str (optional)
+        H5PY file driver used to open H5MD file
+    comm : :class:`MPI.Comm` (optional)
+        MPI communicator used to open H5MD file
+        Must be passed with `'mpio'` file driver
+    convert_units : bool (optional)
+        convert units from MDAnalysis to desired units
+    chunks : tuple (optional)
+        custom chunk layout to be applied to the position,
+        velocity, and force datasets. By default, these datasets
+        are chunked in (1, n_atoms, 3) blocks
+    compression : str or int (optional)
+        HDF5 dataset compression setting to be applied
+        to position, velocity, and force datasets. Allowed
+        settings are 'gzip', 'szip', 'lzf'. If an integer
+        in range(10), this indicates gzip compression level.
+        Otherwise, an integer indicates the number of a
+        dynamically loaded compression filter.
+    compression_opts : int or tup (optional)
+        Compression settings.  This is an integer for gzip, 2-tuple for
+        szip, etc. If specifying a dynamically loaded compression filter
+        number, this must be a tuple of values. For gzip, 1 indicates
+        the lowest level of compression and 9 indicates maximum compression.
+    positions : bool (optional)
+        Write positions into the trajectory [``True``]
+    velocities : bool (optional)
+        Write velocities into the trajectory [``True``]
+    forces : bool (optional)
+        Write forces into the trajectory [``True``]
+    timeunit : str (optional)
+        option to convert values in the 'time' dataset to a custom unit,
+        must be recognizable by MDAnalysis
+    lengthunit : str (optional)
+        option to convert values in the 'position/value' dataset to a
+        custom unit, must be recognizable by MDAnalysis
+    velocityunit : str (optional)
+        option to convert values in the 'velocity/value' dataset to a
+        custom unit, must be recognizable by MDAnalysis
+    forceunit : str (optional)
+        option to convert values in the 'force/value' dataset to a
+        custom unit, must be recognizable by MDAnalysis
+    author : str (optional)
+        name of the author of the file
+    author_email : str (optional)
+        email of the author of the file
+    creator : str (optional)
+        software that wrote the file [``MDAnalysis``]
+    creator_version : str (optional)
+        version of software that wrote the file
+        [:attr:`MDAnalysis.__version__`]
+
+
+    Note
+    ----
+    Parellel writing with the use of a MPI communicator and the ``'mpio'``
+    HDF5 driver is currently not supported.
```

Comment:
Move to after the synopsis/before Parameters. If it doesn't format as a box, use
```reST
.. note::

    Parallel writing ...
```